src/scripts.js

```diff
@@ -1,77 +1,95 @@
+import './images/star-icon.png';
+import Recipe from './classes/Recipe';
+import RecipeRepository from './classes/RecipeRepository';
+import ingredientsData from './data/ingredients';
+import recipeData from './data/recipes';
 
+// global variables
+const recipeRepo = new RecipeRepository(recipeData);
 
+const recipeSection = document.getElementById('allRecipes');
+const modalSection = document.getElementById('recipeModalBackground');
+const filterDropdown = document.getElementById('filterDropdown');
+const searchBar = document.getElementById('searchBar')
 
+//event listeners
+window.addEventListener('load', () => createRecipeCards(recipeRepo.recipes));
+recipeSection.addEventListener('click', createRecipeModal);
+modalSection.addEventListener('click', collapseRecipe);
+filterDropdown.addEventListener('click', filterRecipes);
 
-const recipeNames = [
-    'Loaded Chocolate Chip Pudding Cookie Cups',
-    'Maple Dijon Apple Cider Grilled Pork Chops',
-    "Dirty Steve's Original Wing Sauce",
-    'Elvis Pancakes',
-    'Mock Udi’s Gluten Free Whole Grain Bread',
-    'Ambrosia Cupcakes',
-    'Creamy Coconut Yogurt Bowl with Chocolate Granola (Video)',
-    'Sesame Cookies',
-    'Thai Chicken Tenders with Broiled Pineapple Slaw',
-    'Egg and Rapini Casserole',
-    'Pulled Pork',
-    'Pumpkin Cheesecake Breakfast Smoothie',
-    'Cinnamon Raisin Overnight French Toast w/ Apple Filling',
-    'Brown Butter Garlic Shrimp',
-    'Baked Stuffed Artichokes',
-    'Puppy Chow Cookies',
-    'Easy Creamy Potato Salad with Yogurt',
-    'Slow-Cooker Italian-Stuffed Peppers',
-    'Whole Wheat Milk and Honey Sandwich Bread',
-    'The Ultimate Healthy Soft & Chewy Pumpkin Chocolate Chip Cookies',
-    'Butterscotch Pudding',
-    'quinoa tabbouleh',
-    'Birthday Cake Jello Shots',
-    'Artichoke Spinach Dip Stuffed Mushrooms',
-    'Baked Manicotti with Sausage and Peas',
-    'Spinach Artichoke Quinoa Casserole',
-    'Double Raspberry Soufflés',
-    'A Cake To Warm Any Heart – Banana Split Cake',
-    'Avocado Chickpea Salad',
-    'Slow Cooker Peach Cobbler',
-    '4 Cheese White Pizza',
-    'Cereal Marshmallow Bars',
-    'Hummus Deviled Eggs',
-    'Creamsicle Pie',
-    'Smothered Green Beans',
-    'Pear & Walnut Salad with a Pear Vinaigrette',
-    'Mexican Vegetables on Cornbread',
-    'Barbecue Shrimp Stir-Fry',
-    'Clayudas',
-    'Farro with Shiitake Mushrooms and Baby Spinach',
-    'Bang Bang Shrimp with Napa Cabbage Slaw',
-    'Homemade Italian Meatballs',
-    'Curried Strawberry Chicken Salad',
-    'Bacon Wrapped Stuffed Pork Tenderloin',
-    'Rolo Cookie Bars',
-    'Buffalo Chicken Sliders',
-    "Reese's Pieces Peanut Butter Cookies",
-    'Farmer’s Market Flatbread Pizza',
-    'Vegan Lentil Loaf',
-    'Pastry Cream'
-  ]
+// Let's clean this up to be a proper form submission..?
+searchBar.addEventListener('keypress', function (e) {
+  if (e.key === 'Enter') {
+    searchRecipes()
+  }
+});
```

Comment:
I would like to add a submit form or some sort of icon in the search bar to click instead of just relying on the user to hit 'enter' for the search to execute.

Reply:
Yea I think the search bar could definitely be changed to work more smoothly.